### ISSUE TITLE
how i can access to the dictionary with the faces reconized?

### ISSUE THREAD

**Issue by Xaviz10** [SharpSenses/SharpSenses#8]
im doing a domotic project with the intel realsense camera and one of the task is reconize the face of the person at the entrace, but i need to know how is the person and  using "Face.FaceRecognized += (s, e) =>" i just know if the person was before reconized nothing else.

**Comment by andrecarlucci** [comment]
What do you mean by how is the person? A picture of him/her?

**Comment by Xaviz10** [comment]
no, im sorry. who is the person*. I  want to know the userid for a specific face and for all the faces saved , to give a name for each face, or specify the Useid for a new face...

**Comment by andrecarlucci** [comment]
Unfortunately the RealSenseSDK doesn't give us the ability to add extra info to its internal faces database, but:

You can subscribe to the Camera.Face.FaceRecognized event and get the FaceRecognizedEventArgs.
Look for UserId in this object, them save it somewhere with your extra info yourself :)

**Comment by Xaviz10** [comment]
Thank you for your help. Now I've another issue when a try to turn off the camera, I use cam.Dispose() but I'm getting a runtime error "R6025 pure virtual function call".
